virtcontainers/agent.go

```diff
@@ -206,4 +207,10 @@ type agent interface {
 
 	// resumeContainer will resume a paused container
 	resumeContainer(sandbox *Sandbox, c Container) error
+
+	// updateRoutes will tell the agent to update route table for an existed Sandbox.
+	updateRoutes(routes []*grpc.Route) error
+
+	// updateNetwork will tell the agent to update a new nic for an existed Sandbox.
+	updateNetwork(sandbox *Sandbox, endpoint Endpoint) error
```

Comment:
@sboeuf I don't know what I've done... Fixed.
@WeiZhang555 Since updateInterface and updateRoute are split in agent grpc code, maybe it's better to rename updateNetwork to updateInterface instead of merging them?